### DISCUSSION TITLE
Inline try-catch

### DISCUSSION THREAD

**Discussion by GitGuruGangsta** [dotnet/csharplang#6457] [Language Ideas] [answered]
I think it would be a good idea to add a bit of syntactic sugar for less try-catch-code on small lines.
Ignoring an exception:

```
try <execution line> catch _ ; //try and ignore exception
OR
try _ <execution line>; //try and ignore exception
```

Handle exception:
`try <execution line> catch <execute on exception>;`

**Comment by svick** [comment, marked as answer]
This is effectively a duplicate of https://github.com/dotnet/csharplang/discussions/1015 and https://github.com/dotnet/csharplang/discussions/4676.
